mypy/plugins/enums.py

```diff
@@ -46,6 +55,7 @@ def enum_name_callback(ctx: mypy.plugin.AttributeContext) -> Type:
     """
     enum_field_name = _extract_underlying_field_name(ctx.type)
     if enum_field_name is None:
+        # TODO: handle full enums/unions of enums for names
```

Comment:
Maybe worth creating an issue and quoting that issue here, for a more complete description of the "TODO"? Sometimes I have trouble understanding my own one-liner TODOs that I left a couple months ago :)